src/core/thread/network_data_service.cpp

```diff
@@ -195,20 +195,68 @@ Error Manager::GetNextDnsSrpAnycastInfo(Iterator &aIterator, DnsSrpAnycast::Info
 
 Error Manager::FindPreferredDnsSrpAnycastInfo(DnsSrpAnycast::Info &aInfo) const
 {
+    Error               error = kErrorNotFound;
     Iterator            iterator;
     DnsSrpAnycast::Info info;
-    bool                found = false;
+    DnsSrpAnycast::Info maxSeqNumInfo;
```

Comment:
```suggestion
    DnsSrpAnycast::Info maxNumericalSeqNumInfo;
```
nit: to be explicit

I actually had difficulties reading the code for the first time because I thought `maxSeqNumInfo` is the max Seq num by serial number arithmetic. So, better be more explicit?

Reply:
THanks.